{"name": "test_data[1]", "status": "failed", "statusDetails": {"message": "AssertionError: assert 1 == 3", "trace": "new_data = 1\n\n    def test_data(new_data):\n        print(\"测试步骤\")\n>       assert new_data == 3\nE       assert 1 == 3\n\nscripts\\test_fixture_params.py:17: AssertionError"}, "parameters": [{"name": "new_data", "value": "1"}], "start": 1578278406181, "stop": 1578278406181, "uuid": "ea0d6d04-1f66-4f9b-a30f-2e81188ef572", "historyId": "f975f1c8c8ce09339757bdcce8f1d0e0", "testCaseId": "dd465c263251d58de65400669ad5358e", "fullName": "scripts.test_fixture_params#test_data", "labels": [{"name": "parentSuite", "value": "scripts"}, {"name": "suite", "value": "test_fixture_params"}, {"name": "host", "value": "DESKTOP-TJSPRKG"}, {"name": "thread", "value": "11684-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "scripts.test_fixture_params"}]}